action.yaml

```diff
@@ -0,0 +1,196 @@
+name: Create installer
+description: Build the installer from a recipe directory.
+
+inputs:
+  conda-root:
+    description: Location to a pre-installed conda/mamba installation.
+    required: false
+  container-arch:
+    description: The architecture of the Docker image if different from the host.
+    required: false
+  container-image:
+    description: The Docker image used to build the installer.
+    required: false
+  environment-yaml-file:
+    description: |
+      Path to the environment.yaml file used to create the build environment.
+      Takes precedence over envrionemnt-yaml-string.
+      The file must not have the `name` property.
+    required: false
+  environment-yaml-string:
+    description: |
+      Multi-line string containing the contents of the environment.yaml file used to create the
+      build environment. Can be used to include logic and expressions from the calling workflow.
+      The file must not have the `name` property.
+    required: false
+  recipe-directory:
+    description: The directory containing the recipe file.
+    required: true
```

Comment:
I wouldn't call the `construct.yaml` a "recipe". It can be confusing because this terminology is used for conda-build and so on. What about `input-directory` or `construct-directory` instead?